routes/employee_auth.js

```diff
@@ -95,6 +95,45 @@ const create_employee = async (employee) => {
     }
 }
 
+// Function which creates access token
+const create_access_token = (existing_employee) => {
+    const access_token = jwt.sign({ id: existing_employee.employee_id, admin: existing_employee.employee_is_admin }, config.access_token_secret, { expiresIn: '30s' });
+    return access_token;
+}
+
+// Function which adds refresh token to database
+const add_refresh_token = async (refresh_token) => {
+    try {
+        const TABLE_NAME = 'refresh_tokens';
+        await knex(TABLE_NAME).insert(refresh_token);
+    } catch (err) {
+        return err;
+    }
+}
+
+// Function which finds refresh token on database
+const find_refresh_token = async (refresh_token) => {
+    try {
+        const TABLE_NAME = 'refresh_tokens';
+        const COLUMN_NAME = 'refresh_token';
+        const matching_refresh_tokens = await knex(TABLE_NAME).where(COLUMN_NAME, refresh_token).select();
+        return matching_refresh_tokens[0];
+    } catch (err) {
+        return null;
+    }
+}
+
+// Function which delete refresh token on database
+const delete_refresh_token = async (refresh_token) => {
+    try {
+        const TABLE_NAME = 'refresh_tokens';
+        const COLUMN_NAME = 'refresh_token';
+        await knex(TABLE_NAME).where(COLUMN_NAME, refresh_token).delete();
+    } catch (err) {
+        return err;
+    }
+}
```

Comment:
Can we move these functions to a new model?

Reply:
I don't think we have to. Because new model is only needed when it needs a route

Reply:
good point!